src/index.js

```diff
@@ -25,7 +25,13 @@ export {assembleShaders} from './shader-utils';
 // Lib
 export {Layer, CompositeLayer, LayerManager, AttributeManager} from './lib';
 export {COORDINATE_SYSTEM} from './lib';
+// Viewports
+export {default as Viewport} from './lib/viewports/viewport';
+export {default as ViewportMercatorProject} from './lib/viewports/flat-viewport';
+export {default as FlatViewport} from './lib/viewports/flat-viewport';
```

Comment:
We should delete the FlatViewport completely - it is the old 2D viewport from mercator-project. It will only be confusing to have it here. 

Both of the exports and the file itself should be deleted.